README.md

```diff
@@ -43,20 +43,27 @@ Configure plugin:
       <configuration>
         <chartDirectory>${project.basedir}</chartDirectory>
         <chartVersion>${project.version}</chartVersion>
-        <helmRepoUrl>${helm.repo.url}</helmRepoUrl>
-        <helmUploadUrlStable>${helm.repo.url}/stable/api/charts</helmUploadUrlStable>
-        <helmUploadUrlSnapshot>${helm.repo.url}/snapshot/api/charts</helmUploadUrlSnapshot>
+        <uploadRepoStable>
+            <name>stable-repo</name>
+            <url>${repoBaseUrl}/helm-stable</url>
+            <!-- Basic authentication is supported from HELM version >= 2.9 -->
+            <username>foo</username>
+            <password>bar</password>
+        </uploadRepoStable>
+        <uploadRepoSnapshot>
+            <name>snapshot-repo</name>
+            <url>${repoBaseUrl}/helm-snapshots</url>
```

Comment:
Is there no need for authentication on the snapshot repo?

Reply:
This is just an example where I'd like to show that auth is not mandatory.